src/Acuminator/Acuminator.Analyzers/Descriptors.cs

```diff
@@ -91,5 +91,8 @@ internal static class Descriptors
 
 		internal static DiagnosticDescriptor PX1026_UnderscoresInDacDeclaration { get; } =
 		   Rule("PX1026", nameof(Resources.PX1026Title).GetLocalized(), Category.Default, DiagnosticSeverity.Error);
+
+        internal static DiagnosticDescriptor PX1027_PXGraphUsageInDacProperty { get; } =
```

Comment:
@metallmonster we already reserved the "PX1027" diagnostic name

Reply:
@metallmonster "PX1028" reserved too.  "PX1029" is free diagnostic name. https://wiki.acumatica.com/display/ENG/Acuminator%3A+static+code+analysis+and+coloriser+tool+for+Acumatica+Framework

Reply:
@SENya1990 @saloponov thanks, guys!